tests/testthat/test-workflows.R

```diff
@@ -577,3 +577,22 @@ test_that("can get upstream dependencies", {
     depend2 = "example"
   ))
 })
+
+test_that("orderly workflow works with running a report twice", {
+  path <- orderly_prepare_orderly_example("depends", testing = TRUE, git = TRUE)
+  commits <- git_commits(branch = "master", root = path)
+
+  reports <- list(
+    list(name = "depend4",
+         params = list(nmin = 0.5, another_param = "test")),
+    list(name = "depend4",
+         params = list(nmin = 1, another_param = "test"))
+  )
+
+  summary <- workflow_summary(path, reports, "master")
+  expect_equal(summary$reports, reports)
+  expect_equal(summary$ref, "master")
+  expect_equal(summary$missing_dependencies, list(
+    depend4 = list("example", "depend2")
```

Comment:
This is another change which might not have been expected. Previously would have returned this twice i.e.
```
list(
  depend4 = list("example", "depend2")
  depend4 = list("example", "depend2")
)
```
Do we want this to also go down to 1 entry @LekanAnni ?

Reply:
No, 1 entry won't work.

Reply:
This does fix the duplicate. Maybe I not quite understanding the response you want, could you give me a snippet with json input and your expected output json?

Reply:
`reports: [{"name": "reportName", params: {"key": "value"} },
{name: "reportName", params: {"key2": "value2"}]`

This is actually the same format as request param for "reports"

Reply:
Discussed this, we want 1 entry in `missing_dependencies` per input report